### PR TITLE
i18n(pt-BR): Update `reference/overrides`

### PR DESCRIPTION
**Pull request by Fryuni**
#### Description

Update Portuguese translation of overrides reference

### CODE REVIEW

File: docs/src/content/docs/pt-br/reference/overrides.md [hunk 1]
```diff
@@ -134,6 +135,12 @@ Objeto `Date` JavaScript que representa quando a página foi atualizada pela úl
 
 Objeto `URL` para o endereço onde a página poderá ser editada, se ativado.
 
+#### `labels`
+
+**Tipos:** `Record<string, string>`
+
+Um objeto content as strings da UI localizados para a página atual. Veja o guia [“Traduza a UI do Starlight”](/pt-br/guides/i18n/#traduza-a-ui-do-starlight) para uma lista de todas as chaves disponíveis.
```
Comment (delucis) [inline docs/src/content/docs/pt-br/reference/overrides.md:142]: Double checking “content” is correct here and not “contendo” or “que contém”?

```suggestion
Um objeto contendo as strings da UI localizados para a página atual. Veja o guia [“Traduza a UI do Starlight”](/pt-br/guides/i18n/#traduza-a-ui-do-starlight) para uma lista de todas as chaves disponíveis.
```

**Review by delucis — COMMENTED** [review]
Thanks @Fryuni! Just had one grammar question.

File: docs/src/content/docs/pt-br/reference/overrides.md (hunk 1 above)
Comment (Fryuni) [reply docs/src/content/docs/pt-br/reference/overrides.md:142]: Yep, my mistake

**Review by delucis — APPROVED** [review]
Woo, I got lucky and spotted a Portuguese typo 😀 

Thanks again @Fryuni 🙌

**Pull request merged**